### PR TITLE
Add option tag on ItemView.

### PR DESCRIPTION
**Pull request by Unengine**
### Description

SSIA

### How to test

1. Make an elemental equipment.
2. If it has additional stats or skills, check if `ItemView` shows their count.

### Required Reviewers

@planetarium/9c-dev 

### Screenshot
image

### CODE REVIEW

File: nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs [hunk 1]
```diff
@@ -201,5 +210,31 @@ private void UpdateView()
                 }
             }
         }
+
+        protected virtual void SetOptionTag(ItemBase itemBase)
```
Comment (hskim881028) [inline nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:214]: There are two functions that do the same thing.

Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:214]: Are they both in `EquipmentSlot` and `ItemView`? They are separated classes, but they do similar things.
Then how about making `EquipmentSlot` derives `ItemView`?

Comment (hskim881028) [reply nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:214]: How about changing it like this?
```
 public static bool TryGetOptionTag(ItemBase itemBase, out string tag)
        {
            tag = string.Empty;

            if (!(itemBase is Equipment equipment))
            {
                return false;
            }

            var optionCount = equipment.GetOptionCount();
            if (optionCount <= 0)
            {
                return false;
            }

            var sb = new StringBuilder();
            for (var i = 0; i < optionCount; ++i)
            {
                sb.AppendLine("<sprite name=UI_icon_option>");
            }
            tag = sb.ToString();
            return true;
        }
```

Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:214]: Seems like a good idea.

**Review by hskim881028 — APPROVED** [review]

**Pull request merged**